src/sentry/migrations/0407_group_add_sub_status.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.28 on 2023-04-06 21:44
+
+from django.db import migrations
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = True
```

Comment:
You'll need to split this up into separate migrations. If we make this dangerous and add the column, then we'll have code in prod looking for the column but it won't be there.

You can add the column in a not dangerous migration, and the new indexes separately.